app/src/main/java/com/github/sdp/ratemyepfl/database/FirestoreItem.kt

```diff
@@ -0,0 +1,17 @@
+package com.github.sdp.ratemyepfl.database
+
+import com.google.firebase.firestore.DocumentSnapshot
```

Comment:
I think a number of files have unused imports like in these ones

Reply:
Good point, I'll make an optimize import for the next commit.